react/components/SideBar.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import { IconCaretRight } from 'vtex.styleguide'
+import { injectIntl, intlShape } from 'react-intl'
+import OutsideClickHandler from 'react-outside-click-handler'
+import Animation from 'vtex.store-components/Animation'
+import classNames from 'classnames'
+
+const OPEN_SIDEBAR_CLASS = 'vtex-search-result-sidebar-open'
+
+ /* SideBar component */
+class Sidebar extends Component {
```

Comment:
Isn't this component duplicated of another one in storecomponents or minicart? :thinking: 

If so, just import it instead, preventing code dup

Reply:
There is a sidebar on minicart but mine have different structure and design. A future improvement is to create generic sidebar on storecomponents or styleguide to replace all the ones we are using.

Reply:
What's the difference between them? Cant _HOC_ the main `Sidebar` to make the custom behavior?